yal/tests/test_onboarding.py

```diff
@@ -50,6 +50,18 @@ def update_contact(request):
         tstate.requests.append(request)
         return response.json({}, status=200)
 
+    @app.route("/api/v2/fields.json", methods=["GET"])
+    def get_fields(request):
+        return response.json(
+            {
```

Comment:
Is this response body correct? It's different to the fields.json GET [here](https://github.com/praekeltfoundation/vaccine-eligibility/pull/539/files#diff-65eaca890d8cecebf033b77438466aee345340a1ddbb0d0a6a7857bef8b5f3f0R97)

Reply:
I removed this part, I believe it was a previous attempt of mine and is not being used at present.  Will double check during my dev testing once I have this on QA